src/main/groovy/io/seqera/wave/controller/ViewController.groovy

```diff
@@ -74,8 +75,17 @@ class ViewController {
         // create template binding
         final binding = new HashMap(20)
         binding.build_id = result.buildId
-        binding.build_success = result.succeeded()
         binding.build_exit_status = result.exitStatus
+        switch (result.status) {
+            case BuildStatus.SUCCESS:
+                binding.build_success = true
+                break
+            case BuildStatus.FAILURE:
+                binding.build_failed = true
+                break
+            case BuildStatus.RUNNING:
+                binding.build_in_progress = true
+        }
```

Comment:
This likely can be simplified using `build_state` instead of multiple flags

Reply:
HBS does not support compare between values, that's why i have used it 
https://handlebarsjs.com/guide/builtin-helpers.html#if

Reply:
ohh, didn't realised that. Than does not make much sense.